.github/workflows/link-fail-fast.yaml

```
@@ -0,0 +1,32 @@
+name: Links (Fail Fast)
+
+on:
+  pull_request:
+
+jobs:
+  linkChecker:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 2
+
+      - name: 'Get a list of changed files to process'
+        id: changed-files
+        run: |
+          CHANGED_FILES=$(git diff-tree --name-only --diff-filter 'AM' -r HEAD^1 HEAD | sed -z "s/\n$//;s/\n/' '/g")
+          echo "::set-output name=all_changed_files::${CHANGED_FILES}"
+
+      - name: Download Exclude Path
+        run: |
+          curl https://raw.githubusercontent.com/pingcap/docs/master/.lycheeignore --output .lycheeignore
```

```suggestion
          curl https://raw.githubusercontent.com/pingcap/docs/master/.lycheeignore -O
```
https://curl.se/docs/manpage.html#-O